mitosheet/src/mito/types.tsx

```diff
@@ -782,11 +782,11 @@ export interface AnalysisData {
     codeOptions: CodeOptions;
     userDefinedFunctions: FunctionDocumentationObject[];
     userDefinedImporters: UserDefinedImporter[];
-
     importFolderData: {
         path: string,
         pathParts: string[],
-    } | null
+    } | null;
+    theme: MitoTheme | null; 
```

Comment:
TODO: add this to Streamlit, and make it so that if you pass one to Streamlit, it overrules the Streamlit theme. Also add this to the PR description.